klaviyo/api_helper.py

```diff
@@ -212,10 +243,13 @@ def _request(self, method, url, params=None, data=None, request_type=PRIVATE):
                         v1/v2 returns (dict, list).
         """
         self._is_valid_request_option(request_type=request_type)
-        headers = {
-            'Content-Type': 'application/json',
-            'User-Agent': 'Klaviyo-Python/{}'.format(__version__)
-        }
+
+        if headers is None:
```

Comment:
We're going to lose the User-Agent header here for these post requests with this logic